app/models/user.rb

```diff
@@ -2,8 +2,9 @@
 # Fields are all devise settings; most of the methods relate to call list mgmt.
 class User
   include Mongoid::Document
-  include Mongoid::Enum
   include Mongoid::Userstamp::User
+  extend Enumerize
+  extend ActiveModel::Naming
```

Comment:
Slightly squeamish seeing ActiveRecord related stuff sneak in. Mind explaining it? I don't think it will cause a collision with MongoID but it's good to know in general.